inventorhatmini/__init__.py

```diff
@@ -98,15 +98,18 @@ class InventorHATMini():
     def __init__(self, address=IOE_ADDRESS, motor_gear_ratio=50, init_motors=True, init_servos=True, init_leds=True, start_muted=False):
         """ Initialise inventor hat mini's hardware functions
         """
-        self.address = address
+        self.ioe = None
 
         gpiodevice.friendly_errors = True
 
-        # Setup user button
-        self._pin_user_sw = gpiodevice.get_pin(self.PI_USER_SW_PIN, "IHM-SW", INPD)
+        self.address = address
+
+        if ( address == self.IOE_ADDRESS ): # only if using default address, for use with multiple HATs
+            # Setup user button
+            self._pin_user_sw = gpiodevice.get_pin(self.PI_USER_SW_PIN, "IHM-SW", INPD)
 
-        # Setup amplifier enable. This mutes the audio by default
-        self._pin_amp_en = gpiodevice.get_pin(self.PI_AMP_EN_PIN, "IHM-AMP-En", OUTL if start_muted else OUTH)
+            # Setup amplifier enable. This mutes the audio by default
+            self._pin_amp_en = gpiodevice.get_pin(self.PI_AMP_EN_PIN, "IHM-AMP-En", OUTL if start_muted else OUTH)
```

Comment:
Since these are not assigned for the second IHM, I assume that `switch_pressed()`, ` mute_audio()`, and `unmute_audio()` will fail if called? Can you include some handling for this? E.g. set `self._pin_user_sw` and `self._pin_amp_en` to `None` then check for that in all relevant places?